language/polkavm/move-to-polka/src/tools.rs

```diff
@@ -25,66 +30,83 @@ impl PlatformTools {
 
         Ok(())
     }
+
+    pub fn get_runtime(&self, out_path: &PathBuf) -> anyhow::Result<PathBuf> {
+        debug!("building move-native runtime for polkavm in {out_path:?}");
+        println!("building move-native runtime for polkavm in {out_path:?}");
+        let archive_file = out_path
+            .join("riscv32imac-unknown-none-elf")
+            .join("release")
+            .join("libmove_native.a");
```

Comment:
Question: Similar to the question above. Is there any risk with hard-coding a intermediate name here? Could we ever see a collision if we run in parallel.

Reply:
No. In this case its rust crate and definitely needs a stable name.